### ISSUE TITLE
How to use error estimators with block operators

### ISSUE THREAD

**Issue by acfisher** [mfem/web#146]
At the workshop someone asked how to do this a small example of setting up a composite error estimator would be a useful HowTo article.

**Comment by acfisher** [comment]
Yeah this is basically what I was thinking.  Are you interested in cutting this down to the bare bones of the code and writing a HowTo article on this?

**Comment by tangqi** [comment]
Sure. I will provide this HowTo in a week or two. I will likely need your suggestion/help while I am working on it.
